static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js

```diff
@@ -24,7 +24,7 @@ export const acquisitionsBannerFivTemplate = (
         </div>
         <div class="fiv-banner__container">
             <div class="fiv-banner__text">
-                HIHIHI ${params.messageText}${params.ctaText}
+                HIHIHI ${params.messageText}${params.leadSentence}
```

Comment:
@jranks123 is already editing this file so I haven't changed any of the placeholder stuff